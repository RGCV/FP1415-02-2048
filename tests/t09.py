## ========================================================================== ##
## -------------------------------------------------------------------------- ##
##                                  TEST 09                                   ##
## -------------------------------------------------------------------------- ##
## ========================================================================== ##

# -- Reconhecedores: e_tabuleiro, tabuleiro_terminado -- #
T = cria_tabuleiro()
e_tabuleiro(T)
C = cria_coordenada(1, 2)
T = tabuleiro_preenche_posicao(T, C, 2)
e_tabuleiro(T)
e_tabuleiro(C)
C = cria_coordenada(1, 4)
T = tabuleiro_preenche_posicao(T, C, 2)
e_tabuleiro(T)
escreve_tabuleiro(T)
J = pede_jogada()
W
T = tabuleiro_reduz(T, J)
e_tabuleiro(T)
C = cria_coordenada(1, 4)
T = tabuleiro_preenche_posicao(T, C, 2)
e_tabuleiro(T)
tabuleiro_terminado(T)
escreve_tabuleiro(T)
J = pede_jogada()
S
T = tabuleiro_reduz(T, J)
C = cria_coordenada(1, 4)
T = tabuleiro_preenche_posicao(T, C, 2)
escreve_tabuleiro(T)
e_tabuleiro(T)
tabuleiro_terminado(T)