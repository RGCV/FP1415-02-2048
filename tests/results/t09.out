True
True
False
True
[ 0 ] [ 2 ] [ 0 ] [ 2 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
Pontuacao: 0
Introduza uma jogada (N, S, E, W): W
True
True
False
[ 4 ] [ 0 ] [ 0 ] [ 2 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
Pontuacao: 4
Introduza uma jogada (N, S, E, W): S
[ 0 ] [ 0 ] [ 0 ] [ 2 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 4 ] [ 0 ] [ 0 ] [ 2 ] 
Pontuacao: 4
True
False
